docs/docs/articles/run-tests-with-github-actions.md

```diff
@@ -0,0 +1,143 @@
+# Run Tests with GitHub Actions
+
+**Run on Testkube** is a GitHub Action for running tests on the Testkube platform.
+
+Use it to run tests and test suites and obtain results directly in the GitHub's workflow.
+
+## Usage
+To use the action in your GitHub workflow, use kubeshop/testkube-run-action@v1 action. The configuration options are described in the Inputs section and may vary depending on the Testkube solution you are using (cloud or self-hosted) and your needs.
+
+The most important options you will need are test and testSuite - you should pass a test or test suite name there.
+
+### Testkube Cloud
+To use this GitHub Action for the Testkube Cloud, you need to create an API token.
+
+Then, pass the **organization** and **environment** IDs for the test, along with the **token** and other parameters specific for your use case:
+
+```sh
+uses: kubeshop/testkube-run-action@v1
+with:
+  # Instance
+  organization: tkcorg_0123456789abcdef
+  environment: tkcenv_fedcba9876543210
+  token: tkcapi_0123456789abcdef0123456789abcd
+  # Options
+  test: some-test-id-to-run
+  ```
+
+It will probably be unsafe to keep this directly in the workflow's YAML configuration, so you may want to use [GitHub's secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) instead.
+
+```sh
+uses: kubeshop/testkube-run-action@v1
+with:
+  # Instance
+  organization: ${{ secrets.TkOrganization }}
+  environment: ${{ secrets.TkEnvironment }}
+  token: ${{ secrets.TkToken }}
+  # Options
+  test: some-test-id-to-run
+  ```
+
+### Self-hosted Instance
+
+
+To run the test on self-hosted instance, simply pass url that points to the API, i.e.:
```

Comment:
`url` here may be either `url` as a parameter name, or URL as an acronym itself. Shouldn't it be highlighted or capitalized?
```suggestion
To run the test on self-hosted instance, simply pass **url** that points to the API, i.e.:
```
```suggestion
To run the test on self-hosted instance, simply pass URL that points to the API, i.e.:
```

Reply:
@aabedraba Which  formatting applies to this statement?

Reply:
URL in caps